src/ui/class-classic-editor.php

```diff
@@ -188,7 +188,10 @@ public function add_check_changes_link( $post = null ) {
 		if ( ! \is_null( $post ) && $this->permissions_helper->is_rewrite_and_republish_copy( $post ) ) {
 			?>
 				<div id="check-changes-action">
-					<a href="<?php echo \esc_url( $this->link_builder->build_check_link( $post ) ); ?>"><?php \esc_html_e( 'Do you want to compare your changes with the original version before merging?', 'duplicate-post' ); ?>
+					<?php echo \esc_html_e( 'Do you want to compare your changes with the original version before merging? Please save any changes first.', 'duplicate-post' ); ?>
+					<br><br>
+					<a class='button' href=<?php echo \esc_url( $this->link_builder->build_check_link( $post ) ); ?>>
+						<?php echo \esc_html_e( 'Compare', 'duplicate-post' ); ?>
```

Comment:
Same as above: `esc_html_e` already echoes. The redundant `echo` should be removed.